### PR TITLE
Backup/upgradeable contract hr

### PR DESCRIPTION
**Pull request by hermannleboss**
# Description

Make the Contract upgradable, and deploy it. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by hermannleboss — COMMENTED** [review]
LGTM

File: contract/.openzeppelin/sepolia.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      "layout": {
+        "solcVersion": "0.8.24",
+        "storage": [
+          {
+            "label": "numSodas",
+            "offset": 0,
+            "slot": "0",
+            "type": "t_uint256",
+            "contract": "VendingMachineV1",
+            "src": "contracts\\VendingMachineV1.sol:9"
```
Comment (farrellh1) [inline contract/.openzeppelin/sepolia.json:28]: why there is numSodas in here?

**Review by farrellh1 — APPROVED** [review]
LGTM

File: contract/.openzeppelin/sepolia.json (hunk 1 above)
Comment (hermannleboss) [reply contract/.openzeppelin/sepolia.json:28]: This was something i was trying with another contract. I have to remove it.

**Review by dasarathg68 — APPROVED** [review]

**Pull request merged**